config/checkstyle/checkstyle.xml

```diff
@@ -174,10 +171,6 @@
         </module>
         <module name="NonEmptyAtclauseDescription" />
         <module name="JavadocTagContinuationIndentation" />
-        <module name="SummaryJavadoc">
-            <property name="forbiddenSummaryFragments"
-                      value="^@return the *|^This method returns |^A [{]@code [a-zA-Z0-9]+[}]( is a )" />
-        </module>
```

Comment:
http://checkstyle.sourceforge.net/releasenotes.html#Release_5.7#Release_8.1

> SummaryJavadoc: no violation on absent summary and on javadoc where '.' is used not as end of sentence. Author: Sagar #3907

I removed it from out checkstyle config, since some times our Javadocs start with a sentence ending with `:`. For example: [here](https://github.com/commercetools/commercetools-sync-java/blob/master/src/main/java/com/commercetools/sync/categories/CategorySync.java#L330) also several other places.